proposals/required-members.md

```diff
@@ -263,42 +263,131 @@ init_argument_initializer
 A type with a parameterless constructor that advertises a _contract_ is not allowed to be substituted for a type parameter constrained to `new()`, as there is no way
 for the generic instantiation to ensure that the requirements are satisfied.
 
+### Overriding, Hiding, and Inheriting
+
+It is an error to mark a member required if the member is not at least as visible as the containing member's type. This means the following cases are not allowed:
+
+```cs
+interface I
+{
+    int Prop1 { get; }
+}
+public class Base
+{
+    public virtual int Prop2 { get; }
+
+    protected required int _field; // Error: _field is not at least as visible as Base. Open question below about the protected constructor scenario
+    protected Base() { }
+}
+public class Derived : Base, I
+{
+    required int I.Prop1 { get; } // Error: explicit interface implementions cannot be required as they cannot be set in an object initializer
```

Comment:
>object initializer [](http://example.com/codeflow?start=124&length=18)

Not blocking this PR: I feel like the spec is missing a section on the caller side of the story, ie object initializers ;-) 
We should talk about nested initializers (difference between `{ x = new Point { ... } }` and `{ x = { ... } }`).

Reply:
Added a question for this.